src/cli.py

```diff
@@ -1,30 +1,37 @@
-#!usr/bin/env python3
-"""
-ImageGoNord, a converter for a rgb images to norththeme palette.
+# -*- coding: utf-8 -*-
+#! usr/bin/env python3
+"""ImageGoNord, a converter for a rgb images to norththeme palette.
 Usage: gonord [OPTION]...
 
 Mandatory arguments to long options are mandatory for short options too.
 
 Startup:
-  -h, --help                       print this help and exit
-  -v, --version                    display the version of Image Go Nord and exit
+  -h,  --help                       print this help and exit
+
+  -v,  --version                    display the version of Image Go Nord and
+                                    exit
 
 Logging:
-  -q, --quiet                      quiet (no output)
+  -q,  --quiet                      quiet (no output)
 
 I/O Images:
-  -i=FILE, --img=FILE              specify input image name
-  -o=FILE, --out=FILE              specify output image name
+  -i=FILE,  --img=FILE              specify input image name
+
+  -o=FILE,  --out=FILE              specify output image name
 
 Theme options:
   --PALETTE[=LIST_COLOR_SET]        the palette can be found on the
-                                    src/palettes/ directory by replace the palette
-                                     with the name is possible to select the theme
-                                     and if necessary you can specify the set of
-                                     colors you want to use.
-                                     Ex: python src/cli.py --nord=Aurora,Frost
+                                    src/palettes/ directory (actually there is
+                                    only nord), by replace the palette with the
+                                    name is possible to select the theme and if
+                                    necessary you can specify the set of colors
+                                    you want to use.
+                                    Ex: python src/cli.py --nord=aur,p,s is
```

Comment:
Didn't work:
![image](https://user-images.githubusercontent.com/12409541/166200680-00d8342c-e529-4664-9f7a-bb582a862aec.png)

We might think about removing it instead of fixing it, maybe this is not a required feature. What do you think?

Reply:
I have already removed this feature and updated the documentation in the new client version.

Reply:
src/cli.py is not the new cli?

This is the file: https://github.com/Schrodinger-Hat/ImageGoNord/blob/264eb4ab3f3a6e263345630a78a17bd669832438/src/cli.py